crates/metadata-server/src/grpc/handler.rs

```diff
@@ -53,6 +60,9 @@ impl MetadataStoreHandler {
 #[async_trait]
 impl MetadataServerSvc for MetadataStoreHandler {
     async fn get(&self, request: Request<GetRequest>) -> Result<Response<GetResponse>, Status> {
+        counter!(METADATA_SERVER_GET_TOTAL).increment(1);
```

Comment:
it would have been nice if the counter has a dimension for status (success vs. failure). Imagine wanting to monitor success rate of a give metadata server, how would we do that?

This comment applies to all fallible operations.

Reply:
Yeah, that's definitely a good idea. I will update it